test/Microsoft.AspNet.Authentication.Test/OpenIdConnect/OpenIdConnectAuthenticationHandlerForTestingAuthenticate.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.AspNet.Authentication.Tests.OpenIdConnect
     /// <summary>
     ///  Allows for custom processing of ApplyResponseChallenge, ApplyResponseGrant and AuthenticateCore
     /// </summary>
-    public class OpenIdConnectAuthenticationHandlerForTestingAuthenticate : OpenIdConnectAuthenticationHandler
+    public class OpenIdConnectAuthenticationHandlerForTestingAuthenticate : OpenIdConnectHandler
```

Comment:
Ugh, on a side note, at some point are you going to nuke these classes?  They really don't seem necessary, the middleware exists to replace the logger (instead of just putting the right logger in the container)